src/adapters/mod.rs

```diff
@@ -16,7 +16,7 @@ pub mod tts;
 mod ip_camera;
 
 /// An adapter dedicated to the Philips Hue
-mod philips_hue;
+pub mod philips_hue;
```

Comment:
You're right, it's no longer required since I switched all local ```use``` declarations to use ```super::```. Fixed.